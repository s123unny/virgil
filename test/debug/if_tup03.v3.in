b ../core/if_tup03.v3 6
run
bt
info l
c
q
