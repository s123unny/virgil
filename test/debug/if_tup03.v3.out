breakpoint|../core/if_tup03.v3 6|0
stop|stopOnBreakpoint
bt|main|../core/if_tup03.v3|6
variable|0|a|0|int|false
variable|1|t|(0, 1)|(int, int)|false
stop|end
result|1
